### DISCUSSION TITLE
请问可否增加个对文件夹扫描时候的，监测文件夹里面的歌曲是否有任何变动从而不扫描那些没变动的文件夹，不然每次添加个新的文件夹后都要重新扫描所有的文件夹，尽管这个文件夹并没有改变任何歌曲！

### DISCUSSION THREAD

**Discussion by Andreerassel** [Ferry-200/coriander_player#144] [Ideas]
请问可否增加个对文件夹扫描时候的，监测文件夹里面的歌曲是否有任何变动从而不扫描那些没变动的文件夹，不然每次添加个新的文件夹后都要重新扫描所有的文件夹，尽管这个文件夹并没有改变任何歌曲！
如下图所示，我修改了第二个文件夹的歌曲内容，但是他仍然扫描了第一个文件夹。如果是本地硬盘的话，扫描速度快还行，但是网络硬盘访问会十分慢，120首歌曲每次需要6分钟以上！还望新增该功能，万分感谢！
![屏幕截图 2024-10-17 204739](https://github.com/user-attachments/assets/8db8a259-366b-4249-867c-6d39384cd19b)
![屏幕截图 2024-10-17 204744](https://github.com/user-attachments/assets/e7929f70-8293-4981-873d-7c4491555747)

**Comment by Ferry-200** [comment]
对于本地的文件夹更新，一直都会跳过没有变动的文件夹。我现在根本就没有做网盘读取，所以你通过挂载等方式导入肯定会出现问题。
